.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish new version
+on:
+  push:
+    branches:
+      - master
+
```

Comment:
In some repos we'll also run this on `num.x` and `num.num.x` branches too, like this:

```yml
on:
  push:
    branches:
      - master
      - '[0-9]+.x'
      - '[0-9]+.[0-9]+.x'
```

Those are the branch names sementic-release expects for maintenance branch releases.

Reply:
I removed that because I was pretty sure we don't do maintenance branches of any kind and would only support the latest mainline package.

Reply:
On second thought, I'll add them back just so we're not restricted from doing it in the off chance we need to.

Reply:
My vote is to not support maintenance branches until we need it (YAGNI) to reduce complexity and unexpected behaviour.

Reply:
We have never needed or supported maintenance branches in the past, so that's a good point that it's unlikely we're going to need it in the foreseeable future. We always want people using the latest version of this tool.